cmds/http-gateway/main.go

```diff
@@ -176,16 +179,37 @@ func myHTTPError(ctx context.Context, mux *runtime.ServeMux, marshaler runtime.M
 	}
 	w.Header().Set("Content-Type", contentType)
 
-	body := &errorBody{
-		Error:   s.Message(),
-		Message: s.Message(),
-		Code:    int32(s.Code()),
-		Details: s.Proto().GetDetails(),
-	}
+	var buf []byte
+	var merr error
+	if s.Code() == errors.MissingOVNs {
```

Comment:
oof.. that sucks we have to do this.. might be worth filing a feature request to grpc-gateway folks. dug through their code a little bit, looks like they intentionally completely drop the response message.

Reply:
I'm not sure what the feature request would be; when a gRPC function has an error, it returns a Status rather than the normal response.  It's basically a return type of `StatusOr<ResponseType>`.  It seems like of course the response message is dropped when there is an error; what else would make sense to do?

Reply:
ah, i thought it was sending both, nevermind then